### DISCUSSION TITLE
Paswort für Webzugriff vergessen

### DISCUSSION THREAD

**Discussion by Gambrius** [brutella/hkknx-public#182] [Q&A] [answered]
Ich habe mein Passwort für den Webzugriff vergessen, bzw. bin mir auch nicht sicher, ob ich hier überhaupt ein Passwort gesetzt hatte.
Gibt es ein Standard Passwort, welches genutzt wird, wenn man dieses nicht geändert hat?

Oder gibt es eine Möglichkeit über SSH (der Zugriff funktioniert) das Passwort zurück zu setzen?

**Comment by brutella** [comment, marked as answer]
Du kannst das Passwort löschen, wenn du SSH Zugriff auf den Computer hast.

1. Verbinde dich mit dem Computer über SSH
Wenn du das vorgefertigte Raspberry Pi Image verwendest, dann ist der Benutzer “admin” und das Passwort "homekitbridgeforknx".
Siehe https://hochgatterer.me/de/hkknx/docs/install/rpi-img/

2. Stoppe die Ausführung von hkknx mit dem Befehl `sv stop hkknx`
3. Entferne das Passwort aus der Datenbank Datei, welche unter dem Pfad `/var/lib/hkknx/data/server.json` zu finden ist. Suche den Eintrag `password` und löschen diesen aus der Datei.
4. Starte hkknx neu mit dem Behelf `sv start hkknx`

**Reply by Gambrius** [reply]
Thank you for your response. It worked like a charme.

**Reply by 71z0** [reply]
Hi, `sv stop hkknx` produces a `warning: hkknx: unable to open supervise/ok: access denied` here. What am I doing wrong?

**Reply by brutella** [reply]
@71z0 `access denied` means that you don't have permissions to stop the `hkknx` process. You will need to prefix the command with `sudo` to run it as superuser like this `sudo sv stop hkknx`.
